src/librustc_mir/dataflow/impls/borrows.rs

```diff
@@ -284,18 +524,24 @@ impl<'a, 'gcx, 'tcx> Borrows<'a, 'gcx, 'tcx> {
     }
 
     /// Add all borrows to the kill set, if those borrows are out of scope at `location`.
-    ///
-    /// `is_activations` tracks whether we are in the Reservations or
-    /// the ActiveBorrows flow analysis, and does not set the
-    /// activation kill bits in the former case. (Technically, we
-    /// could set those kill bits without such a guard, since they are
-    /// never gen'ed by Reservations in the first place.  But it makes
-    /// the instrumentation and graph renderings nicer to leave
-    /// activations out when of the Reservations kill sets.)
+    /// That means either they went out of either a nonlexical scope, if we care about those
+    /// at the moment, or the location represents a lexical EndRegion
     fn kill_loans_out_of_scope_at_location(&self,
                                            sets: &mut BlockSets<ReserveOrActivateIndex>,
-                                           location: Location,
-                                           is_activations: bool) {
+                                           location: Location) {
+        /*
+         XXX: bob_twinkles  reintroduce this
```

Comment:
ah, maybe you *don't* yet have `EndRegion`-based support working after all? (or maybe I'll see somewhere else in the PR that they get supported in some manner... I'm going through the PR commit-by-commit at the moment.)